src/Lidar.cc

```diff
@@ -460,4 +461,16 @@ bool Lidar::IsActive() const
   return true;
 }
 
-IGN_SENSORS_REGISTER_SENSOR(Lidar)
+// Register as LidarSensor to conform to naming requirements
+IGNITION_ADD_PLUGIN(
+    ignition::sensors::SensorTypePlugin<Lidar>,
+    ignition::sensors::SensorPlugin)
+IGNITION_ADD_PLUGIN_ALIAS(
```

Comment:
why are you doing this only for this plugin ?

Reply:
I was trying something before but this isn't needed anymore. Fun fact: `Lidar` is the only sensor class that doesn't end in `Sensor`, so I was working around this. But it isn't needed anymore, removed: ecf87cffa2fbe4eef2744aa1f1a4320219658ba2